### PR TITLE
v1.1.2

### PR DESCRIPTION
**Pull request by jamcry**
- update readme
- bump up package version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,40 +36,65 @@ function LoginForm() {
...
+### Production Build
 
-You can start to development with `npm run dev` command. The command watches for changes and builds the toolkit. If you want to generate a production ready build you can use `npm run build`.
+The production deployment is automated by GitHub Actions. Check the [.github/workflows/new-version.yml](/.github/workflows/new-version.yml) file for more information.
 
-Or you can run `npm run storybook` to see the components live. Storybook has own Webpack config that compiles and runs the components.
+---
 
-### Linter
+If you need to generate a production ready build for some reason, use:
```
Comment (yasincaliskan) [inline README.md:94]: Maybe we can also briefly tell our `pre-commit` process and husky.

**Review by yasincaliskan — APPROVED** [review]
It looks perfect @jamcry , thanks for your valuable contributions. We are grateful to you! ❤️ 

Will we move this package to under a new `bakers` organization in `npm`?

**Review by tatata96 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jamcry) [reply README.md:94]: 👍 Yes we can mention it, I'll do it after moving to bakers

**Comment by jamcry** [comment]
> It looks perfect @jamcry , thanks for your valuable contributions. We are grateful to you! ❤️
> 
> Will we move this package to under a new `bakers` organization in `npm`?

hey @yasincaliskan , I will move the repo under bakers' github today, but about npm, for now we decided to keep under hipo organization

**Pull request merged**